sink_service/source/config.c

```diff
@@ -529,7 +554,7 @@ static const sd_bus_vtable config_vtable[] = {
     SD_BUS_WRITABLE_PROPERTY("NetworkAddress", "u", network_add_read_handler, network_add_write_handler, 0, 0),
     SD_BUS_WRITABLE_PROPERTY("NetworkChannel", "y", network_channel_read_handler, network_channel_write_handler, 0, 0),
     SD_BUS_WRITABLE_PROPERTY("SinkCost", "y", sink_cost_read_handler, sink_cost_write_handler, 0, 0),
-    SD_BUS_WRITABLE_PROPERTY("ChannelMap", "u", channel_map_read_handler, channel_map_write_handler, 0, 0),
```

Comment:
What happened to channel_map_read_handler is it used elsewhere in the code ? Delete it if it is not used anymore

Reply:
channel_map_read_handler is declared as a Macro shared with many other settings.
That's why I introduced a wrapper around it.
And it is used in the wrapper of course. I could expend the Macro directly in wrapper but then it would just be code copy/paste